WooCommerce/Classes/Authentication/Store Creation/Installations/StoreCreationSuccessView.swift

```diff
@@ -0,0 +1,127 @@
+import SwiftUI
+
+/// Hosting controller that wraps the `StoreCreationSuccessView`.
+final class StoreCreationSuccessHostingController: UIHostingController<StoreCreationSuccessView> {
+    private let onContinue: () -> Void
+
+    init(siteURL: URL,
+         onContinue: @escaping () -> Void) {
+        self.onContinue = onContinue
+        super.init(rootView: StoreCreationSuccessView(siteURL: siteURL))
+
+        rootView.onContinue = { [weak self] in
+            self?.onContinue()
+        }
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureNavigationBarAppearance()
+    }
+
+    /// Shows a transparent navigation bar without a bottom border and with a close button to dismiss.
+    func configureNavigationBarAppearance() {
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithTransparentBackground()
+        appearance.backgroundColor = .systemBackground
+
+        navigationItem.standardAppearance = appearance
+        navigationItem.scrollEdgeAppearance = appearance
+        navigationItem.compactAppearance = appearance
+    }
+}
+
+struct StoreCreationSuccessView: View {
+    /// Set in the hosting controller.
+    var onContinue: () -> Void = {}
+
+    /// URL of the newly created site.
+    let siteURL: URL
+
+    // Tracks the scale of the view due to accessibility changes.
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    @State private var isPresentingWebview: Bool = false
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
```

Comment:
Suggestion: Instead of using this extra `VStack` to wrap the `ScrollView` and the bottom view, you can also [pin the bottom view](https://nilcoalescing.com/blog/PinAViewToTheBottomOfSafeArea/) to the bottom of the safe area like so:

```Swift
ScrollView {
    // some content here...
}
.safeAreaInset(edge: .bottom) {
    // bottom view here...
}

```

Reply:
I was going to try this after reviewing your PR 😄 this works like a charm, updated in https://github.com/woocommerce/woocommerce-ios/pull/8141/commits/a696db64eb6fa76daa45fa652a68fad01b278cbb